README.md

```diff
@@ -78,11 +78,15 @@ Toute la documentation sur les composants de Système de design GC est accessibl
 ## Installation locale
 
 - Copiez le référentiel `git clone https://github.com/cds-snc/gcds-components`.
-- Allez dans le répertoire `/packages/web`.
 - Exécutez ensuite `npm install` pour installer toutes les dépendances Node.js.
-- Finalement, exécutez `npm run build` pour compiler les composants Web.
+- Finalement, exécutez `npm run build` pour compiler les composants Web. [FR]
+- Finalement, exécutez `npm run build` pour compiler les trois paquets (composants web, react et angular). [FR]
```

Comment:
Finalement, exécutez `npm run build` pour compiler les trois paquets (composants Web, React et Angular).

(it was alright, I only added capital letters)

Reply:
Thank you, appreciate it!

Reply:
Line 82 (Exécutez npm run build pour compiler les composants Web) will be removed and replaced with new text including three packages.